botorch/models/approximate_gp.py

```diff
@@ -386,6 +371,72 @@ def __init__(
 
         self.to(train_X)
 
+    def init_inducing_points(
+            self,
+            inputs: Tensor,
+    ) -> Tensor:
+        r"""
+        Reinitialize the inducing point locations in-place with the current kernel
+        applied to `inputs`.
+
+        Args:
+            inputs: input_data (*batch_shape, n, d)
+
+        Returns:
+            (*batch_shape, m, d) selected inducing point locations
+        """
+
+        with torch.no_grad():
+            num_inducing = self.model.variational_strategy.base_variational_strategy.inducing_points.size(-2)
+            inducing_points = _select_inducing_points(inputs, self.model.covar_module, num_inducing,
+                                                      self._input_batch_shape)
+            self.model.variational_strategy.base_variational_strategy.inducing_points.copy_(
+                inducing_points.clone()
+            )
```

Comment:
wondering if we should reinitialize the variational parameters here as well, since they might be pretty suboptimal for the new inducing point locations.